index.js

```diff
@@ -0,0 +1,63 @@
+const title = document.getElementById('Title');
+const author = document.getElementById('Author');
+const addButton = document.getElementById('AddButton');
+const booksSection = document.getElementById('Books-section');
+
+const keys = Object.keys(localStorage);
+if (localStorage.length > 0) {
+  keys.forEach((key) => {
+    const newTitle = document.createElement('p');
+    newTitle.textContent = JSON.parse(localStorage.getItem(key)).Title;
+    const newAuthor = document.createElement('p');
+    newAuthor.textContent = JSON.parse(localStorage.getItem(key)).Author;
+    const horizontalLine = document.createElement('hr');
+    const removeBtn = document.createElement('button');
+    removeBtn.textContent = 'Remove';
+    removeBtn.classList.add('Remove');
+    removeBtn.setAttribute('id', key);
+
+    removeBtn.addEventListener('click', () => {
+      localStorage.removeItem(removeBtn.id);
+      window.location.reload();
+    });
+
+    booksSection.appendChild(newTitle);
+    booksSection.appendChild(newAuthor);
+    booksSection.appendChild(removeBtn);
+    booksSection.appendChild(horizontalLine);
+  });
+}
+
+addButton.addEventListener('click', () => {
+  if (title.value.length !== 0) {
+    const inputs = {
+      Title: title.value,
+      Author: author.value,
+    };
```

Comment:
This should also be a function, such that you can invoke it whenever you need it.